apps/nextra/pages/en/build/cli/install-cli/install-cli-mac.mdx

```diff
@@ -0,0 +1,36 @@
+import { Callout, Steps } from 'nextra/components'
+
+# Install the Aptos CLI on Mac
+
+For Mac, the easiest way to install the Aptos CLI is with the package manager `brew`.
+
+# Installation
+
+<Steps>
+### Ensure you have `brew` installed https://brew.sh/.
+### Open a new terminal and enter the following commands.
+
+```bash
+    brew update
+    brew install aptos
+```
```

Comment:
For some reason there's an indent for the codeblocks (https://developer-docs-nextra-git-fork-jmintuitive-mig-294fb4-aptoslabs.vercel.app/en/build/cli/install-cli/install-cli-mac). 

Can we do a find and replace for all instances of ```bash under the `cli` folder. And instead make it 

````
```bash filename="Terminal"
````

Note: windows instructions for the CLI would be

````
```bash filename="Powershell"
````

Reply:
For Windows, do this instead, since bash isn't powershell

````
```powershell filename="Terminal"
````

Reply:
I'm not sure if 

````
```powershell
````

has syntax highlighting but down to try!